packages/upload-api/src/test/external-service/index.js

```diff
@@ -65,5 +75,6 @@ export const getExternalServiceImplementations = async (config) => {
     storageProviders,
     blobRetriever,
     router,
+    maxReplicas: 2 // since we have 3 storage nodes, the max replicas is 2
```

Comment:
Seems like a value we'll want to configure in the future, think it's worth making it configurable now?

Reply:
This is for the tests...

Reply:
🤦 sorry. Didn't see that in the file path. Disregard.